distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc

```diff
@@ -1379,6 +1379,483 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 </csw:GetRecordByIdResponse>
 ----
 
+===== `Contextual Operation Examples`
+A distributed query for records with a property name of "title" and a value containing a string like "T#xtToMatch".
+Note: The attribute value singleChar="#" indicates that '#' can be replaced by any other character for the string
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:PropertyIsLike wildCard="*" singleChar="#" escapeChar="!">
+                   <ogc:PropertyName>title</ogc:PropertyName>
+                   <ogc:Literal>T#xtToMatch</ogc:Literal>
+               </ogc:PropertyIsLike>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+<csw:DistributedSearch hopCount="2" /></csw:GetRecords>
+----
+A query for records with a property value equal to "TextToMatch".
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:PropertyIsEqualTo matchCase="true">
+                   <ogc:PropertyName>PropertyName</ogc:PropertyName>
```

Comment:
These queries would serve as better examples if they used a propertyName that is commonly used such as "AnyText"